Makefile

```diff
@@ -157,25 +157,12 @@ ifeq ($(RSA_KEY_LENGTH),8192)
 	CFLAGS += -DTHEMIS_RSA_KEY_LENGTH=RSA_KEY_LENGTH_8192
 endif
 
-DEFAULT_VERSION := 0.11.0
-GIT_VERSION := $(shell if [ -d ".git" ]; then git version; fi 2>/dev/null)
-# check that repo has any tag
-GIT_TAG_STATUS := $(shell git describe --tags HEAD 2>/dev/null)
-GIT_TAG_STATUS := $(.SHELLSTATUS)
-
-ifdef GIT_VERSION
-# if has tag then use it
-        ifeq ($(GIT_TAG_STATUS),0)
-# <tag>-<commit_count_after_tag>-<last_commit-hash>
-                VERSION = $(shell git describe --tags HEAD | cut -b 1-)
-        else
-# <base_version>-<total_commit_count>-<last_commit_hash>
-                VERSION = $(DEFAULT_VERSION)-$(shell git rev-list --all --count)-$(shell git describe --always HEAD)
-        endif
-else
-# if it's not git repo then use date as version
-        VERSION = $(shell date -I | sed s/-/_/g)
-endif
+# Increment VERSION when making a new release of Themis.
+#
+# If you make breaking (backwards-incompatible) changes to API or ABI
+# then increment LIBRARY_SO_VERSION as well, and update package names.
+VERSION = 0.11.1
```

Comment:
We have so many locations to store version number. Couldn't we create one file VERSION in the main directory of the repository with the current version and anywhere it is possible read that value?